clients/src/main/java/org/apache/kafka/clients/admin/KafkaAdminClient.java

```diff
@@ -1122,10 +1108,10 @@ void handleFailure(Throwable throwable) {
 
     @Override
     public ListTopicsResult listTopics(final ListTopicsOptions options) {
-        final KafkaFutureImpl<Map<String, TopicListing>> topicListingFuture = new KafkaFutureImpl<>();
+        final KafkaFutureImpl<Map<String, TopicListItem>> topicListingFuture = new KafkaFutureImpl<>();
```

Comment:
I wanted something that would avoid confusion with `TopicDescription`, which is the output of `describeTopics`.

`describeTopics -> TopicDescription, listTopics -> TopicListing` seems about as intuitive as it gets, right?